src/map.cpp

```diff
@@ -1863,10 +1865,10 @@ void ServerMap::endSave()
 
 bool ServerMap::saveBlock(MapBlock *block)
 {
-	return saveBlock(block, dbase);
+	return saveBlock(block, dbase, m_map_compression_level);
 }
 
-bool ServerMap::saveBlock(MapBlock *block, MapDatabase *db)
+bool ServerMap::saveBlock(MapBlock *block, MapDatabase *db, int compression_level)
```

Comment:
why does it need to be possible to call saveBlock with a custom compression level?

Reply:
Exactly. I briefly debated changing that. But then I preferred keeping it static.